ivy/functional/frontends/paddle/tensor/attribute.py

```diff
@@ -1 +1,35 @@
 # global
+import ivy
+from ivy.functional.frontends.paddle.func_wrapper import (
+    to_ivy_arrays_and_back,
+)
+from ivy.func_wrapper import (
+    with_supported_dtypes,
+)
+
+
+@with_supported_dtypes(
+    {
+        "2.4.2 and below": (
+            "bool",
+            "int8",
+            "int16",
+            "int32",
+            "int64",
+            "uint8",
+            "uint16",
+            "uint32",
+            "uint64",
+            "bfloat16",
+            "float16",
+            "float32",
+            "float64",
+            "complex64",
+            "complex128",
+        )
+    },
+    "paddle",
+)
```

Comment:
In this case, since the supported dtypes are so many, it would be better in terms of readability to use the `with_unsupported_dtypes` decorator.

Reply:
I shortened the supported dtypes only to those which would be relevant to the function i.e complex dtypes